docs/general/attributes.md

```diff
@@ -209,6 +142,48 @@ if they do not cause breaking changes to HTTP semantic conventions.
 | `ipv6` | IPv6 |
 <!-- endsemconv -->
 
+### `network.peer.*` and `network.local.*` attributes
+
+These attributes identify network peers that are directly connected to each other.
+
+For IP-based communication, `network.peer.address` and `network.local.address` should be IP addresses, Unix domain names, or other addresses specific to network type.
+
+Semantic conventions and instrumentations that populate `address`/`port` pairs
+on `server.*`, `client.*`, `destination.*` or `source.*`, SHOULD only populate the `address`/`port` pair on
+`network.peer.*` and `network.local.*` when they are not the same.
+
+`network.peer.port` and `network.local.port` SHOULD only be populated for families that have notion of port.
+
+_Note: Specific structures and methods to obtain socket-level attributes are mentioned here only as examples. Instrumentations would usually use Socket API provided by their environment or sockets implementations._
```

Comment:
👍 I think this is quite valuable as it provides some unambiguous reference definition